README.md

```diff
@@ -11,9 +11,15 @@
 var fs = require('fs');
 var pdf = require('./lib');
 var html = fs.readFileSync('./test/businesscard.html', 'utf8')
-pdf.create(html, { width: '50mm', height: '90mm'}, function(err, buffer) {
+pdf.create(html, { filename: './businesscard.pdf', width: '50mm', height: '90mm'}, function(err, res) {
   if (err) return console.log(err);
-  fs.writeFile('businesscard.pdf', buffer);
+  console.log(res);
+  /*
+    {
+      filename: './businesscard.pdf',
```

Comment:
Oh, we should improve that. The response should be an absolute path. We should parse an input filename to be absolute. e.g. call `options.filename = path.resolve(options.filename)` in the `create` method.